hypershift-operator/controllers/nodepool/nodepool_controller.go

```diff
@@ -531,8 +530,11 @@ func (r *NodePoolReconciler) reconcile(ctx context.Context, hcluster *hyperv1.Ho
 			}
 		}
 	case hyperv1.KubevirtPlatform:
-		// TODO nargaman
```

Comment:
is this commit undoing a change introduced by another commit from this PR?

Reply:
In the first commit (https://github.com/openshift/hypershift/pull/779/commits/6397e09be20c6c2889a2f08dab30a330fc0d72a1) only added the KubeVirtPlatform flag, so I added comment this place to add the platform implementation
In commit https://github.com/openshift/hypershift/pull/779/commits/c60e76296487dc27e1da254601d4c164a12a50f4 I added the relevant code so removed this comment